tests/Block/AuditBlockServiceTest.php

```diff
@@ -43,19 +43,12 @@ protected function setUp()
      */
     public function testExecute()
     {
-        $block = $this->prophesize(Block::class);
-        $revision = $this->prophesize(Revision::class);
```

Comment:
I'd thought you were going to do `$revision = new Revision()`, why did you remove it?

Reply:
My bad, I understood `avoid mocking` as removing, will be fixed.